test/e2e/framework/test_context.go

```diff
@@ -319,6 +343,8 @@ func RegisterCommonFlags(flags *flag.FlagSet) {
 
 	flags.StringVar(&TestContext.SnapshotControllerPodName, "snapshot-controller-pod-name", "", "The pod name to use for identifying the snapshot controller in the kube-system namespace.")
 	flags.IntVar(&TestContext.SnapshotControllerHTTPPort, "snapshot-controller-http-port", 0, "The port to use for snapshot controller HTTP communication.")
+
+	flags.Var(&stringArrayValue{&TestContext.EnabledVolumeDrivers}, "enabled-volume-drivers", "Comma-separated list of volume drivers which are disabled by default. An exmaple is gcepd; see test/e2e/storage/in_tree_volumes.go for full details.")
```

Comment:
wording suggestion: "Comma-separated list of intree volume drivers to run tests against".

Reply:
done